requirements.txt

```diff
@@ -0,0 +1,67 @@
+Babel==2.3.4
+Beaker==1.8.1
+bleach==1.5.0
+blinker==1.4
+-e git+https://github.com/ckan/ckan.git@638262b96feef48e56da0d95f8dd3686092c5ae3#egg=ckan
```

Comment:
Would we expect a specific version of CKAN to be installed?

Also, if I were to be installing this on my server to actually deploy, wouldn't this be already installed? Perhaps a version other than the one used here? Meaning there would be a clash with the requirements.txt.